Changelog.md

```diff
@@ -4,6 +4,10 @@ Language Features:
 
 
 Compiler Features:
+ * Commandline Interface: Add ``--ast-compact-json`` option with Yul input (e.g. with ``--strict-assembly``, ``--yul``)
+ * Commandline Interface: Add ``--ir-ast-json`` and ``--ir-optimized-ast-json`` output options which output the Yul-IR and the optimized Yul-IR in AST compact JSON format.
+ * Standard JSON Interface: Support "irAstJson" amd "irOptimizedAstJson" options in output selection which output the Yul-IR and the optimized Yul-IR in AST compact JSON format.
```

Comment:
```suggestion
 * Standard JSON Interface: Support ``irAstJson`` amd ``irOptimizedAstJson`` options in output selection which output the Yul-IR and the optimized Yul-IR in AST compact JSON format.
```

Reply:
Thanks for the changes @GiokaMarkella, in our regular meeting we decided that the `irAstJson` and `irOptimizedAstJson` should be renamed to `irAst` and `irOptimizedAst` respectively. I.e., just remove the `Json` part.

Could you please made the change? Then the PR should be ready to be merged :)